fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -1112,6 +1135,14 @@ class BrowserRobot {
             SettingsRobot().interact()
             return SettingsRobot.Transition()
         }
+
+        fun clickSurveyButton(interact: BrowserRobot.() -> unit): BrowserRobot.Transition {
```

Comment:
- `-> unit` - should be written with capital 'U' letter
- `BrowserRobot.Transition` - could remove BrowserRobot. as it is redundant